src/components/organisms/AdminVenueView/AdminVenueView.scss

```diff
@@ -18,34 +18,36 @@
     display: flex;
     width: 100%;
     flex-direction: row;
-    justify-content: space-between;
   }
+}
 
-  &__tab {
-    position: relative;
-    display: flex;
-    flex-grow: 1;
-    text-align: center;
-    align-self: center;
-    cursor: pointer;
-    color: rgba($white, 0.6);
-    text-decoration: none;
-
-    @media (max-width: $media-breakpoint--lg) {
-      width: 50%;
-    }
-
-    @media (max-width: $media-breakpoint--md) {
-      width: 100%;
-    }
-
-    &:hover {
-      color: $white;
-    }
-
-    &--selected {
-      color: $white;
-      background-color: $primary;
-    }
+.ViewTab {
```

Comment:
All of the styles should be following BEM and should be inside the container/wrapper styles, because this might override or collide with another style with the same name, imported from another .scss file.

This is also 'hidden' and people won't understand that it exists as a className, if we create a component ViewTab and it has this classname it will collide.

Moving it back inside then wrapper with `__` prefix would be the way to go to keep it encapsulated and name specific.

Reply:
@denisdimitrov fixed in 10746811a3fd6df74bf802630125e13acc06a33a